snake.js

```diff
@@ -287,9 +284,15 @@
       stopMotion({horizontal:false,length: -dy*4});
       y-=dy*5;
     }
-
+    if(lives === 0){
+      gameOver();
+    }
+  }
+  function gameOver(){

```

Comment:
Good idea for resetting the state, but there are more states in the game to keep track of & not just lives & score, Like the direction of the snake, no of snake blocks.

I think we can keep the reloading logic for now, but an `isGameOver`  boolean check with `lives === 0` should suffice.

Reply:
You can move all game data you need to reset after game is over to `gameData` object for example, for easy reset in future.

Reply:
Yeah, good idea.